pom.xml

```diff
@@ -58,15 +58,17 @@
   </scm>
 
   <properties>
-    <revision>5.0.1</revision>
+    <revision>6.0.0</revision>
     <changelist>-SNAPSHOT</changelist>
     <!-- Character set tests fail unless file.encoding is set -->
     <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+    <java.version>17</java.version>
```

Comment:
This is having no effect. Should be `<maven.compiler.release>17</maven.compiler.release>`.